### ISSUE TITLE
[CAIA A11y Research]: MHV Medical Records (August EOM & Sept. 2024)

### ISSUE THREAD

**Issue by AnneCostello** [department-of-veterans-affairs/va.gov-team#90780]
> [!NOTE]
> **High-Level Overview**: `MHV Medical Records`
> - **Research Timeframe**: August 27 – September 5, 2024
> - **Time of Day:** `TBD`
> - **CAIA AT Pilot**:   #91078 Team is scheduling internal pilots [per research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-08-usability-testing-rd3-at/research-plan.md#timeline--screen-reader-users).
> - **Assistive Tech Users**: `8` AT Users (recruiting `16`)
>     - Source of Truth: [DSVA Slack Canvas Link](https://dsva.slack.com/docs/T03FECE8V/F07J9CSDVDY)
> - **Artifacts Available**: 8/16/2024
> - **Testing Environment:** Live Environment [per Methodology](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-08-usability-testing-rd3-at/research-plan.md#methodology) via **[CodeSpace](https://glorious-sniffle-rqjx676prgx2wjqx-3001.app.github.dev/my-health/medical-records/?postLogin=true)** (choose "yes" to trust inquiry, if prompted)
> - **Assistive Tech Being Used**: 
>     - Screen Readers (`8`) Magnifiers/Zoom (`5`) Text-to-Speech (`3`)
> - **Task Lists**: See task lists for next steps/definition of done

## CAIA Assignee(s)

<details><summary>Toggle to view information about the CAIA Team.</summary>

- @rileyorr 
- CAIA a11y @coforma-jamie (backup support)
- CAIA a11y @sara-amanda (ticket refinement and backup support)

</details>

## About the Team

<details><summary>Toggle to view information about the team.</summary>

- Team name: MHV on VA.gov - Medical Records
- OCTO-DE product owner: Marci McGuire
- Product manager: Anne Costello Kennedy
- Designer:
- FE engineer: 
- Accessibility specialist: Riley Orr
- Product/team Slack channel: #mhv-medical-records

</details>

<details><summary>Toggle to view more intake research ticket information</summary>

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### If you have an accessibility champion, what's their name?

Riley Orr

### Calendar checking

- [X] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [X] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [X] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

</details>

## Research Sessions

August 27 - September 5, 2024

> [!TIP]
> - Providing CAIA A11ys with the time slots in advance for AT users, will allow a11ys to avoid conflicts on the CAIA A11y Capacity Calendar, and reach out to other a11ys, should CAIA not be available. 
> - Providing CAIA A11ys with the technology and devices the users will be testing with, will allow CAIA to prepare in advance for the sessions. 

<details><summary>Toggle to view research timing information.</summary>

### Timeframe

- **CAIA Pilot Session**:  #91078

### Length of sessions

- **Session length:** `90` minutes
- **Buffer time between sessions:** `30` minutes
- **Maximum sessions per day:** `4`

</details>

###  Testing Schedule

CAIA A11Ys will be looking for device and AT information. We realize that some of this information will not be available until the day of the session. 

[MHV Medical Records Source of Truth for Scheduling](https://dsva.slack.com/docs/T03FECE8V/F07J9CSDVDY)

Please toggle and use the following table to enter the following information for each participant: Date and Time, Assistive tech (AT), Device and Operating System, Browser, and Skill level:

<details><summary>Toggle to view the AT testing schedule</summary>

P# | Date and Time (ET) | AT | Device and OS | Browser | Skill | CAIA A11ys Available
------------------|------------------|--------------|--------------|--------------|--------------|--------------
✈️ PILOT SESSION (with AT user) | 8/20/2024 1 - 2:30 p.m. ET | JAWS | Desktop; Windows | Edge | Advanced | @rileyorr (co-pilot)
P1 | 8/27/2024 11 a.m. - 12:30 p.m. ET | VoiceOver | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @rileyorr 
P3 | 8/28/2024 11 a.m. - 12:30 p.m. ET| NVDA | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @rileyorr 
P5 | 8/28/2024 3-4:30 p.m. ET | JAWS | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @rileyorr 
P6 | 8/29/2024 1-2:30 p.m. ET | VoiceOver | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @rileyorr 
P9 | 8/30/2024 11a.m. - 12:30 p.m. ET | TalkBack | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @rileyorr 
P10 | 9/3/2024 | TTS - Apple Voice Control | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | @rileyorr 
P# | #/#/24, #:##-#:## #m | VoiceOver, JAWS, etc `TBD` | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | `TBD- Based on a11y availability`
P# | #/#/24, #:##-#:## #m | VoiceOver, JAWS, etc `TBD` | Mobile or Desktop; Windows, MacOS, iOS, Android, etc. `TBD` | Safari, Chrome, etc. `TBD` | Novice, Intermediate, Advanced `TBD` | `TBD- Based on a11y availability`

- **Consideration**: You can also consider asking participants more specific AT and device combinations. 
- **For instance:** TalkBack on Samsung Galaxy 8, or VoiceOver on iPad with magnification.

</details>

**_CAIA A11ys will review the coverage needed, and self-assign, based on a11y availability, with @rileyorr taking priority._**

## Supporting artifacts 

<details><summary>Toggle to view the supporting artifacts</summary>

- [Product outline](Insert URL here between the parenthesis) - `PENDING`
- [User flows](Insert URL here between the parenthesis) - `PENDING`
- [Prototype or mockups](https://glorious-sniffle-rqjx676prgx2wjqx-3001.app.github.dev/my-health/medical-records/?postLogin=true) - `CODESPACE`
- [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-08-usability-testing-rd3-at/research-plan.md)
- [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-08-usability-testing-rd3-at/conversation-guide.md) 
- [Epics/issue that may be helpful](Insert URL here between the parenthesis)
</details>

## Next Steps: Definition of Done

```[tasklist]
### CAIA A11Y Tasks
- [x] **INTAKE TICKET:** @sara-amanda to process intake ticket
- [x] **INITIAL MEETING:** Schedule meeting with team - @rileyorr (CAIA a11y MHV Lead)
- [x] **CALENDAR HOLDS** Update A11y [CAIA Capacity Calendar](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) with holds for user research sessions with AT, when dates become available.
- [ ] https://github.com/department-of-veterans-affairs/va.gov-team/issues/91078
- [x] **REMOVE CALENDAR HOLDS** Remove holds on A11y [**CAIA Capacity Calendar**](https://calendar.google.com/calendar/u/0/embed?src=c_f982350fc73a8162a1088ab14d00dd37335b2b559a60d6b67eb8021416f81c2a@group.calendar.google.com&ctz=America/Chicago) and replace with invitations from project team (or Perigean)
- [x] **RESEARCH NOTES** Create [CAIA notes document](https://docs.google.com/document/d/1cK53wHySjxPqfELMVZR5NJjPmMXsqsLoFU9tk-nE_9A/edit?usp=sharing) - `8/26/2024`
- [x] **Divvy user research session support with @rileyorr and additional CAIA A11ys as needed** - based on availability and technology available to support. @rileyorr plans to do the SR sessions, and @coforma-jamie and @sara-amanda are available as backup.
- [x] **Review research folder and assets**
- [x] **Participate in research** **Date Span:**`8/27/2024` and `9/5/2024`
- [x] **SESSION NOTES**: Use [CAIA notes document](https://docs.google.com/document/d/1cK53wHySjxPqfELMVZR5NJjPmMXsqsLoFU9tk-nE_9A/edit?usp=sharing) - during the sessions
- [x] **SYNC:** Connect with CAIA A11ys on research findings (meeting) with @rileyorr
- [x] **Create/update the CAIA A11y Research Deliverable** This is dependent on the number of study participants CAIA supports. If minimal, A11Ys may share feedback only in the feedback Mural or this ticket.
- [ ] **Share the completed deliverable** with the project team via Slack: `Primary Team Slack` and `Ticket`
```

```[tasklist]
### VFS TEAM Tasks
- [x] Tag @Naomi Eke in Slack [#sitewide-content-accessibility-ia](https://dsva.slack.com/archives/C01K37HRUAH)
- [x] **PROTOTYPE:** Provide prototype to CAIA A11ys via this ticket in the supporting artifacts section 8/16/2024 via Slack - ss adding.
- [x] **DATES NEEDED** See `Table` in the ticket above. Please provide date range - `RCVD` -  and slots - `PENDING` - for AT sessions being planned.
- [x] **ASSISTIVE TECH. INFO NEEDED:** Please provide the AT the users will be utilizing during the research sessions in `Table` in the ticket above.
- [x] **AT Info:** AT Browser, skill level, years of use for AT to be asked during sessions
- [x] **INVITES:** Forward invites from Perigean to CAIA A11ys (Sara and/or Riley)
```

**Comment by sara-amanda** [comment]
## Scheduling

- Adding @coforma-jamie  to the ticket, since she mentioned in the Slack thread that she would be available to assist @rileyorr with sessions, as needed. I am available as well. 
- Will you be using a Canvas for scheduling @AnneCostello ?

**Comment by rileyorr** [comment]
Following up to add that the updated schedule can be accessed in [the AT study Canvas](https://dsva.slack.com/docs/T03FECE8V/F07J9CSDVDY). 

I've also put together [a notes document](https://docs.google.com/document/d/1cK53wHySjxPqfELMVZR5NJjPmMXsqsLoFU9tk-nE_9A/edit?usp=sharing) for the study. The team had mentioned they only need me for the SR sessions so I added myself to those. I'll double check to determine if CAIA folks are open to attend the other AT sessions or if the team would rather limit the number of observers.

**Comment by rileyorr** [comment]
The [notes document](https://docs.google.com/document/d/1cK53wHySjxPqfELMVZR5NJjPmMXsqsLoFU9tk-nE_9A/edit?usp=sharing) has been updated to reflect the last couple sessions. My feedback was shared with the team after each session in each feedback channel thread. @sara-amanda let me know if there's anything else I need to do prior to closing this ticket! Ty
